ui/components/multichain/global-menu/global-menu.js

```diff
@@ -189,30 +191,31 @@ export const GlobalMenu = ({ closeMenu, anchorElement, isOpen }) => {
       borderStyle={BorderStyle.none}
       position={PopoverPosition.BottomEnd}
     >
-      <>
-        <MenuItem
-          iconName={IconName.Notification}
-          onClick={() => handleNotificationsClick()}
-        >
-          <Box
-            display={Display.Flex}
-            flexDirection={FlexDirection.Row}
-            alignItems={AlignItems.center}
-            justifyContent={JustifyContent.spaceBetween}
+      {basicFunctionality && (
```

Comment:
We are hiding notifications when basic functionality is disabled. We can rethink this in a future  PR to re-engage users to notifications.

Reply:
Yes, I agree. I'm not a big fan of the idea of completely hiding the functionality. It might be more interesting to follow the flow used for notifications: if the user wants to enable profile sync and basic functionality is turned off, then show a modal that informs the user that to enable profile sync, basic functionality must also be enabled. By clicking the button on the modal, we can cascade the turning on of basic functionality, authentication, and profile sync.